notebooks/DrizzlePac/sky_matching/sky_matching.ipynb

```diff
@@ -0,0 +1,582 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This is a quick method of opening our different results and gathering the `MDRIZSKY` values: \n",
+    "im = fits.open(\"f160w_globalmin_drz_sci.fits\")\n",
+    "rootname = im[1].data['ROOTNAME']\n",
+    "asn_id = im[1].data['ASN_ID']\n",
+    "orientat = im[1].data['ORIENTAT']\n",
+    "mdrizsky = im[1].data['MDRIZSKY']\n",
+    "im.close()\n",
+    "\n",
+    "im = fits.open(\"f160w_globalmin_match_drz_sci.fits\")\n",
+    "globalmatchrootname = im[1].data['ROOTNAME']\n",
+    "asn_id = im[1].data['ASN_ID']\n",
+    "orientat = im[1].data['ORIENTAT']\n",
+    "globalmatchmdrizsky = im[1].data['MDRIZSKY']\n",
+    "im.close()\n",
+    "\n",
+    "im = fits.open(\"f160w_match_drz_sci.fits\")\n",
+    "matchrootname = im[1].data['ROOTNAME']\n",
+    "asn_id = im[1].data['ASN_ID']\n",
+    "orientat = im[1].data['ORIENTAT']\n",
+    "matchmdrizsky = im[1].data['MDRIZSKY']\n",
+    "im.close()\n",
+    "\n",
+    "im = fits.open(\"f160w_localmin_drz_sci.fits\")\n",
+    "localrootname = im[1].data['ROOTNAME']\n",
+    "asn_id = im[1].data['ASN_ID']\n",
+    "orientat = im[1].data['ORIENTAT']\n",
+    "localmdrizsky = im[1].data['MDRIZSKY']\n",
+    "im.close()\n",
+    "\n",
+    "mdrizsky_val = pd.DataFrame(\n",
+    "    {'Rootname': rootname,\n",
+    "     'globalmin': mdrizsky,\n",
+    "     'globalmin_match': globalmatchmdrizsky,\n",
+    "     'match': matchmdrizsky,\n",
+    "     'local': localmdrizsky})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "index = mdrizsky_val.index.tolist()\n",
+    "globalmin = list(mdrizsky_val['globalmin'])\n",
+    "globalmin_match = list(mdrizsky_val['globalmin_match'])\n",
+    "match = list(mdrizsky_val['match'])\n",
+    "local = list(mdrizsky_val['local'])\n",
+    "\n",
+    "#Plotting code: \n",
+    "fig = plt.figure(figsize=[10,10])\n",
+    "plt.scatter(index, globalmin_match, color='magenta', label='Globalmin + Match')\n",
+    "plt.scatter(index, match, color='navy', label='Match')\n",
+    "plt.scatter(index, local, color='olive', label='Local')\n",
+    "plt.scatter(index, globalmin, color = 'orange', label='Globalmin')\n",
+    "plt.xlabel('Individual Images')\n",
+    "plt.ylabel('MDRIZSKY Value')\n",
+    "plt.legend(loc=\"center left\")\n",
+    "plt.xticks(index)\n",
+    "plt.tight_layout()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 5. Display the full 'matched' mosaic "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sci=fits.getdata('f160w_match_drz_sci.fits')\n",
+    "fig=plt.figure(figsize=(14,14))\n",
```

Comment:
```suggestion
    "fig = plt.figure(figsize=(14,14))\n",
```

Reply:
I agree with @eteq about the static plot and table. I've removed them for now in the version I'm about to commit, and I've added a line to display the Pandas DataFrame that was generated to mimic the table (sans ORIENTAT and ASN_ID columns). If a user wants to reference the table or plot without running the cells themselves, they will be able to see the compiled version of the notebook on the website. I do think the gif should stay, though.